### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by vanshitalavange**
Created home page, it contains:
- header with logo and nav-links
- banner image
- product cards
- categories cards
- footer

- video : https://user-images.githubusercontent.com/86543154/154140284-9dcf7bbe-eb34-44b0-b8bf-777f731c5713.mp4

Live preview: https://deploy-preview-1--the-organic-store.netlify.app/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,526 @@
...
+              <div class="badge">
+                <i
+                  class="fa fa-shopping-cart header-icon header-badge-icon"
+                ></i>
+                <span class="badge-counter badge-round badge-counter-right"
+                  >0</span
+                >
+              </div>
+              <li class="list-item align-center">
+                <a href="" class="nav-link">Cart</a>
```
Comment (shubhamkumar648) [inline index.html:47]: dont keep href attributes empty use # or link

**Review by shubhamkumar648 — APPROVED** [review]
good use of utils class, the good variable name! little suggestion: check your anchor tag

File: stylesheets/pages/index.css [hunk 2]
```diff
@@ -0,0 +1,310 @@
+@import url("../base/colors.css");
+@import url("../base/variables.css");
+
+.display-none {
+  display: none;
+}
```
Comment (shuntzhere) [inline stylesheets/pages/index.css]: You can move this to your library's utility class and then use it across.

**Review by shuntzhere — APPROVED** [review]
Suggested a minor change, Classnames are easily understandable. Nice use of variables across the CSS file.

File: stylesheets/pages/index.css (hunk 2 above)
Comment (vanshitalavange) [reply stylesheets/pages/index.css]: Actually I wasn't using this class anymore, I forgot to remove it. Changes done, thanks

File: index.html (hunk 1 above)
Comment (vanshitalavange) [reply index.html:47]: changes done

**Pull request merged**